### ISSUE TITLE
Feature Request - Enlarge identification name & link to taxonomy

### ISSUE THREAD

**Issue by DerekSikes** [ArctosDB/arctos#6685]
Previously we could view a catalog record & click the identification (taxon/scientific) name to get to that name in the taxonomy table. This seems gone. Can it be returned?

Ideally this should be the one in the upper left corner:

![Screenshot 2023-08-24 at 3 36 35 PM](https://github.com/ArctosDB/arctos/assets/9099827/6a01d065-4a05-4d96-b972-e6cb73212b7c)

AND.... while we're at this, can the font size for that name be made larger than the the font size for the GUID above it? (Perhaps while shrinking the GUID font size).

Something more like how the MCZ does it?
https://mczbase.mcz.harvard.edu/guid/MCZ:Ent:3144

also... the 'identification' box that has the name & determiner & attributes... is also in need of UI tweeking. The attributes currently take up FAR more space than their relative importance while the actual name of the specimen is tiny... As above, I would prefer the name to be more visible and larger than the attributes. Perhaps just make the name bold font and a size larger so it stands out at least as strongly as the attribute column table headers?

**Priority**
_Please assign a priority-label. Unprioritized issues gets sent into a black hole of despair._
I think UI improvements are high priority, but could be convinced to label this medium

**Comment by dustymc** [comment]
>gone

No, but it's behind the eyeball. (And the link is to a classification, not a taxon.)

<img width="539" alt="Screenshot 2023-08-24 at 17 34 00" src="https://github.com/ArctosDB/arctos/assets/5720791/d0c8395d-0033-45dc-a1d3-734f77fda7fb">


> upper left corner:

That's not a taxon name - that's not a "no" but "I'd need full, realistic, scalable instructions."

**Comment by DerekSikes** [comment]
Hidden behind the eyeball is not good UI - it's not intuitive to new users and old experienced users who have engrained habits like myself wonder why the name can't be linked like it was? If you want to keep that link behind the eyeball then fine but that's not a good reason to not ALSO have the name linked as it was (requiring no extra clicking nor extra learning). Is it? 

And what is not scalable about 'make the name larger' than the GUID?

**Comment by Jegelewicz** [comment]
Id like to request that the taxon name also be a hyperlink to the taxon page. 

The eyeball is fine for the classification, but I think we all expect to be able to click the taxon name and go to the taxon page where I could manage the taxon or see other information about it. This is how the old UI worked - so we aren't asking for anything difficult?

**Comment by dustymc** [comment]
> the taxon name also be a hyperlink to the taxon page

Please see https://github.com/ArctosDB/arctos/issues/6685#issuecomment-1692590045, that doesn't make much sense, I need clear and actionable direction.

**Comment by Jegelewicz** [comment]
@mkoo we discussed this at AWG and agreed this is what we wanted - can you help explain?

**Comment by Jegelewicz** [comment]
Can we get this into an intake form? It is a UI request and more than one issue has been filed about it.

Specific requests for tweaks to the appearance of the identification section of a catalog record

1. We all expect to be able to click the taxon name and go to the taxon page where I could manage the taxon or see other information about it beyond my preferred classification. This is how the old UI worked. (So click Neochthebius vandykei and end up at https://arctos.database.museum/name/Neochthebius%20vandykei) The little eye symbol would continue to serve the function of taking one directly to the preferred classification.
2. The identification attributes currently take up FAR more space than their relative importance while the actual name assigned to the specimen is tiny. We would prefer the name to be more visible and larger than the attributes. See also https://github.com/ArctosDB/arctos/issues/9560

I am ignoring Derek's request about the record header as that seems to require a separate discussion.
